tour/exercise-1.md

```diff
@@ -198,8 +198,7 @@ u:http://oldnews.com/a23 uri hash	a6c4d1f
 
 Now that you have data loading, create an observer that watches the reference
 status column.  This observer should increment word counts when new content is
-referenced and decrement word counts when content is dereferenced.  The
-observer should also delete the content when its dereferenced.
+referenced and decrement word counts when content is dereferenced.  The observer should also delete the content if zero URLs reference this one and when content's URL reference counts goes to zero, if it was processed before then its word counts should be decremented before deleting.
```

Comment:
This new sentence is a bit of a run-on. Perhaps end the sentence after `zero`, and the rest rephrased to be a second sentence?

Reply:
Thank you for reviewing, @ctubbsii. Now I tried to write a bit more clear.